src/ngraph/runtime/ie/ie_executable.cpp

```diff
@@ -113,6 +129,12 @@ runtime::ie::IE_Executable::IE_Executable(shared_ptr<Function> func, string devi
     m_network = InferenceEngine::CNNNetwork(func);
     set_parameters_and_results(*func);
 
+#if defined(NGRAPH_IE_DUMP_GRAPHS)
```

Comment:
I wonder if using an environment variable would be better for this so you don't need to rebuild if you want a graph.

Reply:
Using `env_util.hpp`